config/pdc_discovery.yml

```diff
@@ -1,6 +1,6 @@
 default: &default
   dataspace_url: <%= ENV["DATASPACE_URL"] || "https://dataspace-dev.princeton.edu" %>
-  pdc_describe_rss: <%= ENV["PDC_DESCRIBE_RSS"] || "https://pdc-describe-staging.princeton.edu/describe/works.rss" %>
+  pdc_describe_rss: <%= ENV["PDC_DESCRIBE_RSS"] || "https://pdc-describe-prod.princeton.edu/describe/works.rss" %>
```

Comment:
We change this to production because it has better data than staging and production is now public (i.e. not behind the VPN)